src/EventListener/ReadListener.php

```diff
@@ -64,7 +67,12 @@ public function onKernelRequest(GetResponseEvent $event)
 
         if (null === $filters = $request->attributes->get('_api_filters')) {
             $queryString = RequestParser::getQueryString($request);
-            $filters = $queryString ? RequestParser::parseRequestParams($queryString) : null;
+            if ($queryString) {
+                foreach (RequestParser::parseRequestParams($queryString) as $name => $value) {
+                    $name = $this->nameConverter ? $this->nameConverter->denormalize($name) : $name;
```

Comment:
Wouldn't this apply to every parameters like `itemsPerPage` or custom PropertyFilter / GroupFilter parameter name, and thus create an unexpected app behavior change and/or add more discrepencies between the app and the Swagger documentation ?

Reply:
`itemsPerPage` is indeed not a good example, even if it will be present here, it doesn't matter much.
The point about Group Filter (or any other filter where the name can be defined in the definition) for example is that one could declare explicitly the parameter name as `group_filter_parameter_name` and it might not be respected and transformed to `groupFilterParameterName` for example. I'm not sure it is a  big deal, just wondering if it might be misleading.

Reply:
@antograssiot Mayde do you have a proof test to give me so I can add it on the PR?

Reply:
@Soullivaneuh 
here are the easy downside of your approach:
- First you only normalize keys so ExistsFilter or OrderFilter that uses `foo[paramName]` will not be taken into account
- The hydra:mapping where the filter description leaves doesn't reflect those changes. Maybe updating `ApiPlatform\Core\Hydra\Serializer\CollectionFiltersNormalizer` could fix that

You can use the Dummy entity in the test suite, it has a `$nameConverted` property that is transformed via the `ApiPlatform\Core\Tests\Fixtures\TestBundle\Serializer\NameConverter\CustomConverter`.

You can ping me on slack for this topic if you need quick help/chat